tests/bsim/bluetooth/mesh/_mesh_test.sh

```diff
@@ -3,7 +3,7 @@
 
 source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
 
-EXECUTE_TIMEOUT=600
```

Comment:
Hasn't it been already fixed here: https://github.com/zephyrproject-rtos/zephyr/pull/70087 2 weeks ago?
Does it require more time since that fix?

Reply:
Yep, for the last 2 weeks it grew up in 20 times. It seems weird to me. Does CI become 20 times slower? PC from 90th?

Reply:
pretty much :smile: 
![image](https://github.com/zephyrproject-rtos/zephyr/assets/68428270/38776d14-582c-4da1-8a72-6e26a1c124fb)
https://github.com/zephyrproject-rtos/infrastructure/issues/150

@aescolar correct me if I'm wrong, but it's my understanding it's the parallelization of the runs that is causing the timeouts on those older machines.

Reply:
Ok, since this is watchdog. I close topic to not hair up it. :)